frontend/src/pages/talks/TalkSingleEntry.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { Box, Paper, Typography } from '@mui/material';
+import { TalkEntry } from 'src/services/mocks';
+import { extractVideoId } from 'src/utils/video';
+import { useTranslation } from 'react-i18next';
+
+const TalkSingleEntry = ({ talk }: { talk: TalkEntry; compact?: boolean }) => {
+  const { i18n } = useTranslation();
+
+  const abstractParagraphs = talk.abstract.split('\n');
+
+  function isPast(talk: TalkEntry) {
+    return new Date(talk.date) < new Date();
+  }
+
+  const title = (
+    <Box display="flex" gap={2}>
+      <Typography variant="h4">{talk.title}</Typography>
+    </Box>
+  );
```

Comment:
It looks like we are display this title inside another Typography component.

I suggest to delete this variable and to simply display the `talk.title` inside a single Typography.

Reply:
Yes, my bad